sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgMethodId.cs

```diff
@@ -117,6 +117,9 @@ public static UcfgMethodId CreateMethodId(IMethodSymbol methodSymbol)
         public static UcfgMethodId CreateTypeId(INamedTypeSymbol typeSymbol) =>
             Create(typeSymbol.ConstructedFrom.ToDisplayString());
 
+        public static UcfgMethodId CreateArrayTypeId(IArrayTypeSymbol arrayTypeSymbol) =>
```

Comment:
I find it weird that _CreateTypeId_ and _CreateArrayTypeId_ return a type called "method id".